packages/odata-service-inquirer/src/prompts/datasources/cap-project/cap-helpers.ts

```diff
@@ -33,7 +34,9 @@ async function getCapProjectPaths(
 
     for (const root of capProjectRoots) {
         const folderName = basename(root);
-        capRootPaths.push({ folderName, path: root });
+        // On Windows the path my have been returned with a different casing.
```

Comment:
```suggestion
        // On Windows the path may have been returned with a different casing.
```
small one, typo here?